src/odemis/driver/xt_client.py

```diff
@@ -2140,6 +2172,19 @@ def __init__(self, name, role, parent, hfw_nomag, **kwargs):
             setter=self._setBeamletIndex
         )
 
+        # The compound lens focusing mode allows to switch between immersion mode or not.
+        # So instead of passing a float, we just provide a boolean, with the immersion
+        # mode always set to the same (hard-coded) value. When reading, anything above
+        # 0 is considered in immersion.
```

Comment:
From this comment it's not clear to me what compound lens focusing values 1.0, and 3.0-9.0 are. Can you clarify that? 

In addition to this, does the following comment mean that the compound lens focusing mode (in the adapter) allows some sort of yes no switch, or does that actually allow a certain level of compound lens focusing?
`# The compound lens focusing mode allows to switch between immersion mode or not.`